op-guide/clustering.md

```diff
@@ -109,3 +109,35 @@ We run PD and TiKV on every node and TiDB on node1 only.
     ```sh
     mysql -h 192.168.199.113 -P 4000 -u root -D test
     ```
+    
+## Add/Remove node dynamically
+
+### PD
+
+You can use `join` to start a new PD server. For example, we have started three PD servers within Cluster ID 1:
```

Comment:
You can use `join` to start a new PD server and add it to an existing PD cluster.